wolfcrypt/test/test.c

```diff
@@ -27521,6 +27521,13 @@ static wc_test_ret_t hpke_test_single(Hpke* hpke)
             ret = WC_TEST_RET_ENC_EC(ret);
     }
 
+    if (ret == 0) {
+        ret = wc_HpkeGenerateKeyPair(NULL, &receiverKey, rng);
+        if (ret != -173)
```

Comment:
Please use `BAD_FUNC_ARG` not -173.

Reply:
Please add a code comment indicating this is a negative test case (expected failure).